### PR TITLE
Add the files to the project

### PR DESCRIPTION
**Pull request by Moise-code**
In this project I :

- created a todo list application using the best javasript practices.

### CODE REVIEW

File: src/interactive.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('change', () => {
+    let id;
```
Comment (Victoria129) [inline src/interactive.js:13]: Try to declare variables at the top of your file so they are easy for someone reading your code to see.

File: src/interactive.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    let curStatus;
```
Comment (Victoria129) [inline src/interactive.js]: Try to declare variables at the top of your file so they are easy for someone reading your code to see.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  display: flex;
+  justify-content: space-between;
+  padding: 2% 0;
+  min-height: 3%;
+  border-bottom: 1px solid gray;
+}
+
+.form-input {
+  display: flex;
+  justify-content: space-between;
```
Comment (Victoria129) [inline src/style.css]: Try to use DRY with your code. There are multiple lines of code repeated as "display: flex", "justify-content: space-between"

**Review by Victoria129 — COMMENTED** [review]
- Try to declare variables at the top of your file so they are easy for someone reading your code to see.
- Try to use DRY with your code. There are multiple lines of code repeated as "display: flex", "justify-content: space-between"

File: src/interactive.js (hunk 2 above)
Comment (Moise-code) [reply src/interactive.js]: It make sense, let me fix this thank you.

File: src/style.css (hunk 3 above)
Comment (Moise-code) [reply src/style.css]: Yeah sure, thanks

**Pull request merged**